screens/CameraRecording.tsx

```diff
@@ -49,9 +50,8 @@ const CameraRecording: FunctionComponent = () => {
                   maxFileSize: 150000000,
                   quality: Camera.Constants.VideoQuality["480p"],
                 })
-                .then((vid) => {
-                  console.log(vid.uri);
-                  setRecording(false);
+                .then(async (vid) => {
+                  await uploadFileToFirebase("avocado.mov", vid.uri);
```

Comment:
is this the file name?  if so, why is it a constant (ie, not containing timestamp, or user id, or something). i also saw that it was interpolated in the firebaseInteractor.ts so my question there might be related (in which don't ahve to answer both of them)

Reply:
No, this is just temporary until we confirm the naming scheme with Paul.